GeoBot/GeoBot/Helpers/Token.cs

```diff
@@ -8,6 +8,7 @@ namespace GeoBot.Helpers
     public class Token
```

Comment:
Not sure if the class name `Token` is now good, since there are more details than just the token... but I don't have a better name in mind.. maybe:
```suggestion
    public class SpeechDetails
```

Reply:
I agree...! Actually I don't know what will be the good naming of this helper class! 
This class will be used both "SpeechToken" and "DirectlineToken"! 
### SpeechToken
```json
{ token:"<tokenvalue>", region:"koreacentral:}
```
### DirectlineToken
In case of directlineToken, it doesn't need region, so it will return null value for region.
```json
{ token:"<tokenvalue>", region:null}
```
Do you think is this a good idea to share this token class for two different purpose?

Reply:
If the data delivered is different the structure/classes ideally should also be different as well, this is better for maintainability and extensibility because it will reduce F/U errors made.